spec/requests/v1/qc_reception_spec.rb

```diff
@@ -0,0 +1,266 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe '/qc_receptions' do
+  describe '#post' do
+    before do
+      # QcAssayType for Extraction - to be ignored
+      create(:qc_assay_type, key: 'qubit_concentration_ngul', label: 'Qubit DNA Quant (ng/ul) [ESP1]', used_by: 0, units: 'ng/ul')
+      # Relevant QcAssayType for ToL
+      create(:qc_assay_type, key: 'sheared_femto_fragment_size', label: 'Sheared Femto Fragment Size (bp)', used_by: 1, units: 'bp')
+    end
+
+    # Load config file
+    let(:config) { YAML.load_file(Rails.root.join('config/locales//en.yml'), aliases: true) }
```

Comment:
```suggestion
    let(:config) { YAML.load_file(Rails.root.join('config/locales/en.yml'), aliases: true) }
```
Hm is there a neater way to access the `en.yml` consts?

Reply:
Maybe dot notation?

Reply:
Tried, couldn't get it right with dot notation, so left it as it is.